Packages/com.unity.inputsystem/InputSystem/Actions/Composites/Vector2Composite.cs

```diff
@@ -1,8 +1,14 @@
+using System;
+using UnityEditor;
```

Comment:
@Rene-Damm: At a quick skim, this `using UnityEditor;` looks like it would cause a compile error in builds so should be moved into the `#if UNITY_EDITOR` block below.

Reply:
Good catch. No idea why this didn't break on the farm. Fixed in https://github.com/Unity-Technologies/InputSystem/commit/06d8b42a81d6b83e9926fd83bd05d15456bccea7.